internal_transfers/actions/scripts/backfill.ts

```diff
@@ -46,9 +48,20 @@ if (TENDERLY_ACCESS_KEY && TENDERLY_USER && TENDERLY_PROJECT) {
 }
 
 export const args = parse<RuntimeArgs>({
-  dateFrom: String,
-  dateTo: String,
+  dateFrom: (value: string) => {
+    if (!dateRegex.test(value)) {
+      throw new Error("Invalid date format. Please use the YYYY-MM-DD format.");
+    }
+    return value;
+  },
```

Comment:
As Ben mentioned, create a helper function to avoid duplication

Reply:
Hey @alfetopito and @bh2smith thanks for the suggestions, will incorporate the same!

Reply:
Great this is looking much better! 

Now that we have a nice standalone function, we could probably test it! How would you feel about moving this to `src/util.ts` and adding a couple unit tests to `tests/util.spec.ts`?

Reply:
Sure @bh2smith will do that 👍🏼